DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -60,12 +60,13 @@ public void UpdateLoginCountAndDuration(int progressId)
 		                    FROM Sessions AS S
 		                    WHERE S.CandidateID = Progress.CandidateID
 		                      AND S.CustomisationID = Progress.CustomisationID
-		                      AND (S.LoginTime BETWEEN Progress.FirstSubmittedTime AND Progress.SubmittedTime)),
+		                      AND S.LoginTime >= Progress.FirstSubmittedTime),
                             Duration = (SELECT COALESCE(SUM(S1.Duration), 0)
 		                    FROM Sessions AS S1
 		                    WHERE S1.CandidateID = Progress.CandidateID
 		                      AND S1.CustomisationID = Progress.CustomisationID
-		                      AND (S1.LoginTime BETWEEN Progress.FirstSubmittedTime AND Progress.SubmittedTime))
+		                      AND S1.LoginTime >= Progress.FirstSubmittedTime),
+                            SubmittedTime = GETDATE()

```

Comment:
The old system uses utc date so we should probably use that as well to remain consistent 🙂